fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/BIOAsync3.scala

```diff
@@ -31,5 +38,26 @@ trait BIOAsync3[F[-_, +_, +_]] extends BIO3[F] {
   def parTraverse_[R, E, A, B](l: Iterable[A])(f: A => F[R, E, B]): F[R, E, Unit] = void(parTraverse(l)(f))
   def parTraverseN_[R, E, A, B](maxConcurrent: Int)(l: Iterable[A])(f: A => F[R, E, B]): F[R, E, Unit] = void(parTraverseN(maxConcurrent)(l)(f))
 
+  /**
+    * Returns an effect that executes both effects,
+    * in parallel, combining their results into a tuple. If either side fails,
+    * then the other side will be interrupted.
+    */
+  final def zipPar[R, E, A, R1 <: R, E1 >: E, B](fa: F[R, E, A], fb: F[R1, E1, B]): F[R1, E1, (A, B)] = zipWithPar(fa, fb)((a, b) => (a, b))
```

Comment:
I have noticed that `parTraverseN_` is not `final`. Should this also not be `final`?